### ISSUE TITLE
Plex Plug-ins Issue

### ISSUE THREAD

**Issue by BioRunner** [Cigaras/IPTV.bundle#179]
Hello all,

I'm running Plex Server on a FreeNAS box. I installed the IPTV plugin on my server and it showed up under Channels just fine. I tested some IP TV links and it all was working fine. I started having issues with the plug-in after I put some wrong m3u links in. So I tried to remove the IPTV plugin from Plex and it would not remove it, maybe permissions issue? So I went onto the server and deleted the IPTV.bundle folder. Then I unzipped another copy of it. Went back on my Plex server and it will not show the IPTV plugin under Channels anymore. For the life of me I can't get the IPTV plugin to show up in Channels anymore.

Any idea what is going on?

I have rebooted the Plex server many times, with no love.

Thank you.

**Comment by BioRunner** [comment]
I followed the directions "How to install IPTV.bundle multiple times" in the wiki and renamed my installation to IPTV2.bundle and Plex showed the plugin under Channels after restarting the server. Not sure what was keeping Plex from detecting the normal naming of IPTV.bundle. I recommend to others to not delete the IPTV.bundle folder unless you are sure you don't want to go through what I went through.
